csrc/device_lower/utils.cpp

```diff
@@ -1933,6 +1933,42 @@ Val* proveLinearAndGetStride(
   return proveLinearAndGetStrideAfterPropagation(frontier, domain);
 }
 
+std::unordered_set<ValGroup> projectTo(
```

Comment:
It seems this is quite similar to `getReachableValsFrom`, which I recently added. 

https://github.com/NVIDIA/Fuser/blob/main/csrc/iter_visitor.h#L616

Reply:
Let me try if I can just use `getReachableValsFrom`, if yes, I will remove this utility, if not, I will move it to `val_graph_visitor.h`.

Reply:
`getReachableValsFrom` worked. Removed `projectTo`.

Reply:
cc: @naoyam Looks like `getReachableValsFrom` does not work for multi-tile tests.

Reply:
Do you know why?

Reply:
Didn't look deep, but I believe the only difference between `getReachableValsFrom` and `projectTo` is the traversal direction. I believe generally it is most safe if `from` fully covers `to`. If not, `require_all_to_visited=true` may make some case work, but seems that it is not guaranteed.